### ISSUE TITLE
Can't drag after touch

### ISSUE THREAD

**Issue by splashful** [soundar24/roundSlider#62]
If you touch a point on the slider you are then restricted from dragging unless you do a second touch. I understand the logic here but in some cases this makes operation difficult. Is there a way to enable slide after touch without Second touch?

**Comment by soundar24** [comment]
The mentioned requirement was already considered here: https://github.com/soundar24/roundSlider/issues/55

Also in the next version i planned to include this feature. Currently I have provided the sample level workaround to achieve this. Check the below demo and let me know your comments:

## http://jsfiddle.net/soundar24/utqgufx0/295/

**Comment by splashful** [comment]
Very nice!! That's working great. However, I read #55 several times carefully and have no idea what he's talking about. It seems to be unrelated to my issue.

**Comment by soundar24** [comment]
Actually in https://github.com/soundar24/roundSlider/issues/55, he reported that changing the value by dragging the handle was hard since in mobile devices its difficult to grab the smaller handle. So he requested a behavior like to change the value by dragging anywhere inside the circle.

Here your requirement also, in initial touch the events may not be hook so that it needed the second touch. But by changing the drag area to the whole circle will solve both the problems. This was done in the mentioned samples as workaround, so it might solves your requirement.

If any additional requirement needed with this behavior then you can update that details also..

**Comment by splashful** [comment]
Hi again. In your previous comment, you offered a solution that worked well. However, I have found that the disable method now fails, and does not disable the slider.

$(".slider").roundSlider("disable");

However, when I remove the previous customization, the disable works correctly.

**Comment by soundar24** [comment]
Yes, I can see the issue. Now i have updated the demo to resolve the issue, check the below fiddle:
## http://jsfiddle.net/soundar24/utqgufx0/385/
